tests/acceptance/helpers-test.js

```diff
@@ -33,6 +33,14 @@ describe('Acceptance: helpers', function() {
           expect(fs.existsSync(addonPath)).to.equal(true);
         });
     });
+
+    it('emberNew - extraCliArgs', () => {
+      return emberNew({ extraCliArgs: ['--typescript', '--no-welcome']})
+        .then(() => {
+          const appPath = path.resolve(process.cwd(), 'app');
+          expect(fs.existsSync(appPath)).to.equal(true);
```

Comment:
This doesn't really test the extra option, but I'm not sure how to test it either.

Testing the output of an actual existing cli options seems frail.

Should I try to use some sort of mocking / stubbing of the ember function to see if it's called with the extra arguments we pass?

I found https://github.com/theKashey/rewiremock, which might be useful for that.

Reply:
I tried using testdouble.js since we already depend on that, but I can't get it to work.. I just want to spy on the `ember` function that's used inside `emberNew` but it doesn't work as expected. I assume I'm doing something wrong here but I'm not sure if I want to keep spending time on this either..